test/smoke/wip/deploy_quarkus_with_persistence.feature

```diff
@@ -7,14 +7,16 @@ Feature: Service Deployment: Quarkus with persistence
     Given Deploy quarkus example service "jbpm-quarkus-example" with persistence enabled and native <native>
     And DeploymentConfig "jbpm-quarkus-example" has 1 pod running within <minutes> minutes
     And HTTP GET request on service "jbpm-quarkus-example" with path "orders" is successful within 3 minutes
-    And HTTP POST request on service "jbpm-quarkus-example" with path "orders" and body "{\"approver\" : \"john\", \"order\" : {\"orderNumber\" : \"12345\", \"shipped\" : false}}"
+    And HTTP POST request on service "jbpm-quarkus-example" with path "orders" and "json" body '{"approver" : "john", "order" : {"orderNumber" : "12345", "shipped" : false}}'
+    And HTTP GET request on service "jbpm-quarkus-example" with path "orders" should return an array of size 1 within 1 minutes
     
     When Scale DeploymentConfig "jbpm-quarkus-example" to 0 pods within 2 minutes
```

Comment:
This is not reliable, if operator detects that DC has been scaled manually it will override DC. If operator manage to do it before OpenShift detects that DC has been scaled then respin won't happen.

I think that it would be better to scale DC by setting replicas parameter in CR.